packages/react-core/src/next/components/Select/Select.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { css } from '@patternfly/react-styles';
 import { Menu, MenuContent, MenuProps } from '../../../components/Menu';
-import { Popper } from '../../../helpers/Popper/Popper';
+import { Popper, PopperProps } from '../../../helpers/Popper/Popper';
```

Comment:
Do you think the Select still needs a `minwidth` prop?.

Reply:
Select (and Dropdown) still have access to the `minWidth` prop via `popperProps` that get spread directly to `Popper`.

Reply:
I was just wondering if it would just be spread with `popperProps`.  For The next Dropdown it looks like  `minWidth` was removed on line 26.  We should probably be consistent wit the two components.